packages/data-stores/src/user/controls.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Internal dependencies
+ */
+import { wpcomRequest, WpcomClientCredentials } from '../utils';
+import { CreateAccountAction } from './types';
+
+export default function createControls( clientCreds: WpcomClientCredentials ) {
+	return {
+		CREATE_ACCOUNT: async ( action: CreateAccountAction ) => {
```

Comment:
Looks a bit clumsy, but should we get this from the enum since we're getting all the other type names from the enum?

```suggestion
		[ ActionType.CREATE_ACCOUNT ]: async ( action: CreateAccountAction ) => {
```

Same with the `FETCH_CURRENT_USER` key below.

Reply:
Nice, that makes things a bit more consistent.